src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -732,6 +743,11 @@ private static ImmutableArray<RefKind> GetRefKindsOrNull(ArrayBuilder<RefKind> r
                 }
                 refKinds[p] = argRefKind;
             }
+
+            return;
+
+            bool IsLambdaConversion(BoundExpression expr)
```

Comment:
>IsLambdaConversion [](start = 17, length = 18)

I think our convention is to start local function names with a lower-case letter.